package.json

```diff
@@ -9,6 +9,7 @@
   },
   "scripts": {
     "build": "ember build",
+    "coverage": "pr-bumper check-coverage",
```

Comment:
I find this a little misleading. `npm run coverage` sounds like you're running coverage, not checking it.